pkg/logs/config/config_keys.go

```diff
@@ -263,20 +263,28 @@ func (l *LogsConfigKeys) useV2API() bool {
 	return l.getConfig().GetBool(l.getConfigKey("use_v2_api"))
 }
 
-func (l *LogsConfigKeys) getVectorConfigKey(key string) string {
-	return "vector." + l.vectorPrefix + key
+func (l *LogsConfigKeys) getVectorConfigKey(configPrefix string, key string) string {
```

Comment:
Why keep the name of this function `getVectorConfigKey` instead of changing it to `getObsPipelineConfigKey` (or something similar) to be coherent with the rest of the PR ?